derex/runner/cli.py

```diff
@@ -47,7 +47,8 @@ def ddc_ironwood():
     """
     check_docker()
     setup_logging()
-    if not check_services(["mysql", "mongodb", "rabbitmq"]):
+
+    if not check_services(["mysql", "mongodb", "rabbitmq"]) and sys.argv[1] != "down":
```

Comment:
I think this can be improved in a couple of ways:

* to make the code more explicit I'd factor out the checking of wether the `docker-compose` command was `down` to a function with an explicit name like `command_is_down`
* to make the code a bit less brittle (currently the `down` command has to be issued with no option, and this breaks if the user specifies, for instance, the `--verbose` option) I'd use the check `"down" in sys.argv`

Anyway, I think this should not block this PR and I'll merge it.